src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -133,22 +133,30 @@ const BoxAISidebar = (props: BoxAISidebarProps) => {
         spreadsheetNotice = formatMessage(messages.welcomeMessageSpreadsheetNotice);
     }
 
+    const handleSuggestedQuestionsFetched = (fetchedSuggestedQuestions: SuggestedQuestionType[]) => {
+        setCacheValue('suggestedQuestions', fetchedSuggestedQuestions);
+    };
+
+    const suggestedQuestions = getSuggestedQuestions === null ? localizedQuestions : [];
+
     return (
         // BoxAISidebarContent is using withApiWrapper that is not passing all provided props,
         // that's why we need to use provider to pass other props
         <AgentsProvider value={cache.agents}>
             <BoxAISidebarContext.Provider value={contextValue}>
                 <BoxAISidebarContent
+                    cachedSuggestions={cache.suggestedQuestions}
```

Comment:
After latest update in `content-answers` PR, should this now be `cachedSuggestedQuestions`?